README.md

```diff
@@ -47,6 +47,8 @@
   <summary>Client</summary>
   <ul>
     <li><a href="https://reactjs.org/">React.js</a></li>
```

Comment:
- [ ] Please remove REACTJS, ExpressJs, and Postgress because they are not part of the technologies that you are using for this project.